cmd/tools/jwt/README.md

```diff
@@ -0,0 +1,11 @@
+This command line helps to generate `jwtSecret` and `token` for Jenkins client.
+
+## Usage
+
+Run it without Kubernetes:
+
+`go run jwt_cmd.go -s secret`
+
+Run it in Kubernetes, try to find ConfigMap and update it:
```

Comment:
This command has a default value. See also [here](https://github.com/LinuxSuRen/ks-devops/pull/28/files#diff-0a53f6dd491f12732ff650e50a4f889348e2f4f6d39fa92b7f9aba7bef35b11cR35).

Reply:
Well, I mean, we may need to explain it in the documentation.

Reply:
Actually, users don't need to care about the name of ConfigMap from the CLI side. It can handle all the expected ConfigMap. And perhaps we only use it from the helm chart.